CONTRIBUTING.md

```diff
@@ -42,17 +42,16 @@ As a basic rule, before filing issues, feature requests or anything else, please
 
 #### Node.js, npm, node-gyp
 
-Please make sure you are running the exact node version used by the developers/contributors as specified in the [nvmrc file](./.nvmrc).
+Please make sure you are conforming to the `engines` requirements used by the developers/contributors as specified in the [`package.json`](./package.json#engines) and [`recipes/package.json`](./recipes/package.json#engine) files.
 
-Currently, these are the combinations of system dependencies that work on an intel-based machines for MacOS/Linux/Windows (building on an ARM-based machine is still a work-in-progress due to node-sass native dependencies)
+Currently, these are the combinations of system dependencies that work for MacOS/Linux/Windows:
 
 ```bash
-node -v
-v14.17.3
-npm -v
-6.14.12
-node-gyp -v
-v8.0.0
+$ jq --null-input '[inputs.engines] | add' < ./package.json < ./recipes/package.json
+{
+  "node": "16.14.2",
+  "npm": "8.7.0",
```

Comment:
For building the `recipes` submodule, we also need `pnpm`. Did you run the `jq` command above?

Reply:
this is the output I got
![image](https://user-images.githubusercontent.com/20743451/165317899-c5e2b828-6889-4f83-92fe-a1de9008e837.png)

Reply:
that's coz you do not have the `recipes` submodule checked out. I have done the changes for this